walmart/walmart.py

```diff
@@ -307,6 +307,88 @@ def get_cancel_payload(self, lines):
             ), xml_declaration=True, encoding='utf-8'
         )
 
+    def update_shipment(self, order_id, package):
+        headers = {
+            'Content-Type': "application/xml"
+        }
+        url = self.url + '/%s/shipping' % order_id
+        return self.connection.send_request(
+            method='POST',
+            url=url,
+            body=self.get_shipment_payload(package),
+            request_headers=headers
+        )
+
+    def get_shipment_payload(self, package):
+        """Shipment Update
+
+        :param package: {
+            "tracking_number": "",
+            "tracking_url": "",
+            "carrier": "",
+            "carrier_service": '',
+            "ship_date_time": '',
+            "items": [{
+                "line_number": "",
+                "quantity": 2,
+            }]
+        }
+        """
+        element = ElementMaker(
+            namespace='http://walmart.com/mp/v3/orders',
+            nsmap={
+                'ns2': 'http://walmart.com/mp/v3/orders',
+                'ns3': 'http://walmart.com/'
+            }
+        )
+        return etree.tostring(
```

Comment:
Better split this giant structure into small maintainable chunks.

Reply:
@tarunbhardwaj I disagree. This structure is highly relatable to the original xml schema and makes much more sense. Also the list comprehension is super fast considered to the for loop append. I would say, add tests for this.

Reply:
I guess `package['items']` will never big enough to make performance more priority here over maintainability.

Reply:
Its just that i do not see any issue with maintainability here. This just looks like the xml schema. But dont let my comment prevent this patch from making further changes. I m just a Contributor :)
*peaceout*